docs/user-guide/layer-settings.md

```diff
@@ -248,6 +248,25 @@ The style editor can configure the *Icon* as explained [here](layer-settings.md#
 
 <img src="../img/layer-settings/classify_icon_ex.jpg" class="ms-docimage">
 
+### Create a style for 3D Tiles
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) also allows creating a style fo [3D Tiles layer](catalog.md#3d-tiles-catalog). In this case, the style panel looks like this:
+
+<img src="../img/layer-settings/3dtiles-style-panel.jpg" class="ms-docimage">
+
+The user can **Add a fill rule** by clicking on <img src="../img/button/add_fill_button.jpg" class="ms-docbutton"/> button and customize the `Fill color`.
+
+<img src="../img/layer-settings/ex_3dtiles_style.gif" class="ms-docimage">
+
+Once the *Fill Rule* is added, the editor can:
+
+* **Filter** the style rule by clicking on the <img src="../img/button/filter_white_button.jpg" class="ms-docbutton"/> button in order to apply the style only to certain layer features. 
+
+* **Remove** the symbolizer by clicking the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton"/> button.
+
```

Comment:
For the 3D Tiles styling, while with the Code Text Editor it is possible to leverage completely on the styling specifications:

![image](https://user-images.githubusercontent.com/1280027/164754428-d1ae866e-4e5f-4572-973a-3f1d8ff733ff.png)

the MapStore Visual Style Editor supports for now only a limited set of capabilities:

- Customization of the Fill color

![image](https://user-images.githubusercontent.com/1280027/164752406-943a0b31-eb4b-4845-b080-06401c20f0d9.png)

- Style Rule filtering based on the available [properties dictionary ](https://github.com/CesiumGS/3d-tiles/tree/1.0/specification#properties) defined in the tileset.json

![image](https://user-images.githubusercontent.com/1280027/164752305-ce8b2460-3994-4bbb-81e3-ff27380e8795.png)

- Possibility to customize the radius in case of point cloud features

![image](https://user-images.githubusercontent.com/1280027/164751967-8c0813a1-d291-453f-a4c0-24d99aea87ff.png)

![image](https://user-images.githubusercontent.com/1280027/164752187-dc4d6999-7b1e-4361-bb35-dc9e65ed9378.png)